docs/cmdline-opts/output-dir.d

```diff
@@ -0,0 +1,15 @@
+Long: output-dir
+Arg: <dir>
+Help: Directory to save files in
+Added: 7.72.0
+See-also: remote-name remote-header-name
+---
+
+This option specifies the directory in which files should be stored, when
+--remote-name or --output are used.
+
+The given output directory is used for all URLs and output options on the
+command line, up until the first --next.
+
+If this option is used multiple times, the last specified directory will be
+used.
```

Comment:
Is the directory creation implicit? If not, might be good to add a reference to `--create-dirs`?

Reply:
No, this option doesn't try to create the directory at all. It just prepends the name to the file name used to save the file. I should verify that `--create-dirs` actually work fine together with this...